util/types/range.go

```diff
@@ -22,6 +22,11 @@ import (
 	"github.com/pingcap/tidb/sessionctx/variable"
 )
 
+// Range is the interface of the three type of range.
```

Comment:
1. I think this file can be moved to package util/ranger
2. I think this interface can implement convert2***Range